<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TLSplashScreen</class>
 <widget class="QWidget" name="TLSplashScreen">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>214</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 71, 71);
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="image">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="text">
     <property name="font">
      <font>
       <family>Mukti Narrow</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="font">
      <font>
       <family>Mukti Narrow</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Don't show this message again</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="button0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:20px;
border-style: solid;
border-width: 1px;
color: rgb(255, 255, 255);
font: 10pt &quot;Mukti Narrow&quot;;
padding-left:15px;
padding-right:15px;
}
QPushButton::!pressed {
background-color: rgb(0, 0, 0);
border-color: rgb(170, 170, 255);
}
QPushButton::pressed {
background-color:#444448;
border-color: #fad100;
}</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:20px;
border-style: solid;
border-width: 1px;
color: rgb(255, 255, 255);
font: 10pt &quot;Mukti Narrow&quot;;
padding-left:15px;
padding-right:15px;
}
QPushButton::!pressed {
background-color: rgb(0, 0, 0);
border-color: rgb(170, 170, 255);
}
QPushButton::pressed {
background-color:#444448;
border-color: #fad100;
}</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:20px;
border-style: solid;
border-width: 1px;
color: rgb(255, 255, 255);
font: 10pt &quot;Mukti Narrow&quot;;
padding-left:15px;
padding-right:15px;
}
QPushButton::!pressed {
background-color: rgb(0, 0, 0);
border-color: rgb(170, 170, 255);
}
QPushButton::pressed {
background-color:#444448;
border-color: #fad100;
}</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
